aspnetcore/aspnetcore-2.0.md

```diff
@@ -3,6 +3,7 @@ title: What's new in ASP.NET Core 2.0
 author: rick-anderson
 description: Learn about the new features in ASP.NET Core 2.0.
 manager: wpickett
+monikerRange: '>= aspnetcore-2.0'
```

Comment:
Should this doc ONLY display for 2.0? If so, change this metadata to the following:

```
monikerRange: '= aspnetcore-2.0'
```

Reply:
I was thinking you'd want to show that to the 2.1 folks but maybe not.
If you just got approval to go from 1.1 to 2.1 you might miss it.

Reply:
I suppose that does make sense.